packages/client/webpack.config.ts

```diff
@@ -97,7 +98,8 @@ const config: Configuration = {
         '%%API_HOST%%': isProd ? 'https://api.codementoring.co' : 'http://localhost:4000',
         '%%IS_PROD%%': isProd
       }
-    })
+    }),
+    new copyPlugin( [{ from:'./_redirects', to:'./' }] )
```

Comment:
This definitely goes into the dist folder?

Reply:
Yes, I tested it.  It goes into it.  I find it strange too.  If I add ./dist  it will create dist/dist/_redirects

Reply:
do you want all the package-lock.json delete on root, api and client?

Reply:
Ok, glad that works. We shouldn't have any package-lock.json. It should all be yarn